src/index.tsx

```diff
@@ -17,4 +19,12 @@ const root = document.getElementById('root')
 
 if (!root) throw new Error('No #root element found in the DOM.')
 
-render(() => <App />, root)
+render(
+  () => (
+    <QueryClientProvider client={new QueryClient()}>
```

Comment:
Things in this file don't have any test coverage at the moment because we just import `<Routes />` from `App.tsx` in `App.browser.test.tsx`...

I think if `UploadQueue` was visible on page load it would have caused the test to fail from trying to access the query client?

tl;dr move these to `AppLayout` in `App.tsx` (I just created a PR to add it in #358)